<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceWheelDiameterSetPage</class>
 <widget class="QWidget" name="MaintainceWheelDiameterSetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 50);</string>
  </property>
  <widget class="QPushButton" name="BTN1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN3">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN5">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN8">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN7">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN4">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNHome">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>620</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/Home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN9">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTC1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>MC1车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLMP1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>MP1车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>MC2车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>MP2车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum0">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>530</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNumC">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>530</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum9">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_TC1Value">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 50);</string>
   </property>
   <property name="text">
    <string>770</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_MP1Value">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 50);</string>
   </property>
   <property name="text">
    <string>770</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_M1Value">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 50);</string>
   </property>
   <property name="text">
    <string>770</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_M2Value">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>130</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 50);</string>
   </property>
   <property name="text">
    <string>770</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>122</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 50);</string>
   </property>
   <property name="text">
    <string>轮径
(mm)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTC1SetValue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 100);
border-right:2px solid rgb(170,170,170);
border-bottom:2px solid rgb(170,170,170);
border-radius:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNMP1SetValue">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 100);
border-right:2px solid rgb(170,170,170);
border-bottom:2px solid rgb(170,170,170);
border-radius:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNM1SetValue">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>190</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>3</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 100);
border-right:2px solid rgb(170,170,170);
border-bottom:2px solid rgb(170,170,170);
border-radius:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNM2SetValue">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>190</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>4</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 100);
border-right:2px solid rgb(170,170,170);
border-bottom:2px solid rgb(170,170,170);
border-radius:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>488</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 50);</string>
   </property>
   <property name="text">
    <string>车号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNMetroNumSet">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>270</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>8</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 100);
border-right:2px solid rgb(170,170,170);
border-bottom:2px solid rgb(170,170,170);
border-radius:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNSelectAll">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>7</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 100);
border-right:2px solid rgb(170,170,170);
border-bottom:2px solid rgb(170,170,170);
border-radius:0px;</string>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_seting">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>轮径设定值
770-850mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>31</y>
     <width>31</width>
     <height>69</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border-top-left-radius:8px;
border-bottom-left-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>941</x>
     <y>31</y>
     <width>31</width>
     <height>69</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border-top-right-radius:8px;
border-bottom-right-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>轮径,线路号,车号
需分别设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
